src/EFCore.PG.NTS/Query/ExpressionTranslators/Internal/NpgsqlNetTopologySuiteMemberTranslatorPlugin.cs

```diff
@@ -18,6 +18,24 @@ public class NpgsqlNetTopologySuiteMemberTranslatorPlugin : IMemberTranslatorPlu
 
     public class NpgsqlGeometryMemberTranslator : IMemberTranslator
     {
+        private static readonly CaseWhenClause[] _ogcGeometryTypeWhenThenList = new[]
```

Comment:
nit: omit the implicit `private` modifier for consistency

Reply:
Should be fixed